### ISSUE TITLE
ES6 module needs exporting

### ISSUE THREAD

**Issue by Heverett94** [dambem/ClimateMonitorV2#117]
## Description
ES6 module needs exporting to allow testing. 

## The actual vs expected behaviour
At the moment functions are out of scope and unaccessable i believe this is due to the import statement for guage.js.

**Comment by dambem** [comment]
Yes! This was the issue. I managed to sort of hot-fix it by appending var to necessary functions to force them into a global scope but obviously not the best shot. Thank you for finding this I was wondering what was causing it!

**Comment by Heverett94** [comment]
Might work having it all in a class then define the class as a variable on a global scope. Then export class.blah instead of varing it all?

**Comment by dambem** [comment]
@Megan-Wright @AlvaroRausell

**Comment by AlvaroRausell** [comment]
We had a shot at ES6'ing the gauge code, but the issue is that importing it then to a ES5 codebase (`new_mapping.js`) gave us lots of problems. How necessary are the tests to the gauge?
